packages/terra-application/src/terra-dev-site/test/application-status-overlay/StatusOverlayAllProps.test.jsx

```diff
@@ -14,9 +14,12 @@ const ApplicationStatusOverlayAllProps = () => {
   ];
 
   return (
-    <ApplicationStatusOverlayProvider>
-      <ApplicationStatusOverlay buttonAttrs={StatusViewButtons} message="Status View with all props" variant="no-data" />
-    </ApplicationStatusOverlayProvider>
+    // eslint-disable-next-line react/forbid-dom-props
+    <div style={{ height: '500px' }}>
```

Comment:
should terra-dev-site provide a demo style?

Reply:
It would be impossible for us to guess what hight is appropriate for the example, So i think it's up to the creator of the example.